### ISSUE TITLE
User Story: Create Project structures

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP5_Jeweller#11]
EPIC:  Agile #10 

As a **scrum master/developer** I can **follow Agile principles** so that **project functionality is achieved incrementally across sprints**

### Assumptions or Pre-Requisites:
* Agile principles to be adopted from start of project

### Acceptance Criteria: (Must be completed before task is moved to 'Done')
- [ ] Must have elements setup for the CI_PP5_Jeweller as per tasks listed below

### Tasks

- [x] Task1  Create Github project
- [x] Task2 Setup Sprints/iterations (1 week in duration)
- [x] Task3 Create milestones (First deploy, MVP, Version 1)
- [x] Task4 Setup MoSCoW labels - 01_MustHave, 02_ShouldHave, 03_CouldHave, 04_WontHave: 
- [x] Task5 Setup issue priorities (P1 = highest priority, P2 = medium priority, P3 = lower priority) - these can be set at the start of each sprint
- [x] Task6 Setup Level of Effort labels using T-shirt sizing: S M L XL
- [x] Task7 Setup fields for estimated and actual SPs (these can be observed in sprint review burndown charts)
- [x] Task8 Setup (created, modify) Project Boards to show Kanbans at EPIC & Issue levels with 5 categories - Backlog, InProgress, Review, Done (can add WontDo later) 

Before changing task status to 'Review' or 'Done' please provide comment (and screenprints if appropriate) as documentary evidence of task completion

**Comment by DeeMcCart** [comment]
The project setup probably best illustrated by showing the Kanban board at end of Sprint1:
At this point the project structures are in place and have been used for the first Sprint, there are only a small number of issues on the board as first task for Sprint2 is to add the functional user stories to issues.

Issue/Task View:



EPIC view:
